lib/core/logconf.c

```diff
@@ -786,6 +792,21 @@ char *h2o_log_request(h2o_logconf_t *logconf, h2o_req_t *req, size_t *len, char
             APPEND_DURATION(pos, proxy_total_time);
             break;
 
+        case ELEMENT_TYPE_PROXY_BYTES_WRITTEN_TOTAL:
+            RESERVE(sizeof(H2O_UINT64_LONGEST_STR) - 1);
+            pos += sprintf(pos, "%" PRIu64, (uint64_t)req->proxy_stats.bytes_written.total);
```

Comment:
You would no longer need these type casts once the types are changed to `uint64_t`.